### PR TITLE
testify: More documentation

### PR DESCRIPTION
**Pull request by abhinav**
CC @bombela @prashantv

### CODE REVIEW

**Comment by bombela** [comment]
lgtm

File: testify.go [hunk 1]
```diff
@@ -77,7 +99,50 @@ func (st sinkTestingT) Errorf(format string, args ...interface{}) {
...
+// continue on failure.
+//
+// Require(T): All asserts will result in a success or failure being logged to
+// the crossdock.T. Execution of the behavior will be terminated immediately
+// on failure.
+//
+// Fatals(t): Only failures will be logged to crossdock.T. Execution of the
+// behavior will be temrinated immediately on failure.
+//
+//	                     +--------+--------+---------+--------+
```
Comment (prashantv) [inline testify.go:137]: this table doesn't look very aligned:
image

**Comment by prashantv** [comment]
minor formatting nit, lgtm otherwise

File: testify.go (hunk 1 above)
Comment (abhinav) [inline testify.go:137]: Fixed.

**Pull request merged**